overlay.d/15fcos/usr/lib/systemd/system/coreos-check-wireless-firmwares.service

```
@@ -0,0 +1,10 @@
+# This service is used for printing a message if
+# some wireless firmwares are missing
+[Unit]
+Description=Check if all the wireless firmwares are installed
```

Shouldn't it be "any" instead? Also Title Case.
How about just

```suggestion
Description=Check For Wireless Firmware Packages
```

?